keps/sig-windows/2578-windows-conformance/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: Windows Conformance
+kep-number: 2578
+authors:
+  - "@vladimirvivien"
+  - "@jayunit100"
+  - "@perithompson"
+  - "@johnSchnake"
+  - "@claudiubelu"
+  - "@knabben"
+  - "@rosskirkpat"
+  - "@hxietkg"
+owning-sig: sig-windows
+participating-sigs:
+  - sig-testing
+status: implementable
```

Comment:
@knabben can you set the state to provisional now so we can get this merged?
We'll set to the implementable once we get PRR and everything done.

Reply:
Actually, i think we can merge as is and then make updates if there is anything needed for PRR review.